lib/autocomplete.ts

```diff
@@ -61,9 +53,26 @@ export function autocomplete<T>(
     ...options
   };
 
-  const contentFunc = options.contentFunc!;
+  // Keeps track of the last value as typed by the user.
+  let lastAsTyped = inputElem.value;
+  const lis = dom.onElem(inputElem, 'input', () => lastAsTyped = inputElem.value);
+
+  const contentFunc = () => [
+    dom.autoDispose(lis),
```

Comment:
This doesn't seem right: `lis` is created once for the inputElem, but is disposed every time that popup is closed.